it/controllers/benefits/CompanyVanFuelBenefitsAmountControllerISpec.scala

```diff
@@ -0,0 +1,606 @@
+    val expectedTitle: String = "How much was your client’s total company van fuel benefit?"
+    val expectedHeading: String = "How much was your client’s total company van fuel benefit?"
+    val expectedContent: String = "You can find this information on your client’s P11D form in section G, box 10."
+    val expectedErrorTitle: String = s"Error: $expectedTitle"
+    val wrongFormatErrorText: String = "Enter your client’s company van fuel benefit amount in the correct format"
+    val emptyErrorText: String = "Enter your client’s company van fuel benefit amount"
+    val maxAmountErrorText: String = "Your client’s company van fuel benefit must be less than £100,000,000,000"
+  }
+
+  def benefits(carModel: CarVanFuelModel): BenefitsViewModel = BenefitsViewModel(Some(carModel), isUsingCustomerData = true, isBenefitsReceived = true)
+
+  val userScenarios: Seq[UserScenario[CommonExpectedResults, SpecificExpectedResults]] = {
+    Seq(UserScenario(isWelsh = false, isAgent = false, CommonExpectedEN, Some(ExpectedIndividualEN)),
+      UserScenario(isWelsh = false, isAgent = true, CommonExpectedEN, Some(ExpectedAgentEN)),
+      UserScenario(isWelsh = true, isAgent = false, CommonExpectedCY, Some(ExpectedIndividualCY)),
+      UserScenario(isWelsh = true, isAgent = true, CommonExpectedCY, Some(ExpectedAgentCY)))
+  }
+
+  ".show" should {
+    userScenarios.foreach { user =>
+      import Selectors._
+      import user.commonExpectedResults._
+      import user.specificExpectedResults._
+
+      s"language is ${welshTest(user.isWelsh)} and request is from an ${agentTest(user.isAgent)}" should {
+        "render the company van fuel benefits amount page without pre-filled form" which {
+
+          lazy val result: WSResponse = {
+            dropEmploymentDB()
+            userDataStub(userData(fullEmploymentsModel(hmrcEmployment = Seq(employmentDetailsAndBenefits(fullBenefits.map(_.copy(benefits = fullBenefits.flatMap(_.benefits.map(_.copy(mileage = None))))))))), nino, taxYearEOY)
+            insertCyaData(employmentUserData(isPrior = true, cyaModel("name", hmrc = true, Some(benefits(fullCarVanFuelModel.copy(vanFuel = None))))), userRequest)
+            authoriseAgentOrIndividual(user.isAgent)
+            urlGet(url(taxYearEOY), welsh = user.isWelsh, headers = Seq(HeaderNames.COOKIE -> playSessionCookies(taxYearEOY)))
+          }
+
+          implicit def document: () => Document = () => Jsoup.parse(result.body)
+
+          "has an OK status" in {
+            result.status shouldBe OK
+          }
+          titleCheck(user.specificExpectedResults.get.expectedTitle)
+          h1Check(user.specificExpectedResults.get.expectedHeading)
+          captionCheck(expectedCaption)
+          textOnPageCheck(get.expectedContent, contentSelector)
+          textOnPageCheck(amountHint, hintTextSelector)
+          textOnPageCheck(poundPrefixText, poundPrefixSelector)
+          inputFieldCheck(amountInputName, inputSelector)
+
```

Comment:
you could add a check for the input field value to show that it's not pre-filled